### PR TITLE
Added a pick and place that is related to the Y1 demo simulation scenario

### PR DESCRIPTION
**Pull request by felipebelo**
Added anew 'tutorial' called SecondHandsY1PickAndPlace. 

Before running the PickAndPlaceScenario (or the other ones)
(scenarios/PickAndPlaceScenario/startScenario.sh) you will need to run the
Y1DemoSimulationScenario. That's
the simulation scenario with the Y1 scene.

The Y1DemoSimulationScenario will be added to the Github repositories soon.

### CODE REVIEW

**Comment by mirkow** [comment]
Please dont commit the build directory and temp files

**Comment by MuriloMartins** [comment]
Thanks for looking into that @mirkow !
@felipebelo just use the same `.gitignore` file from the other repos...

**Comment by felipebelo** [comment]
I added .gitignore and removed build and temporary files.

File: SecondHandsY1PickAndPlace/README.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+THIS REPOSITORY IS TEMPORARY AND WILL BE MOVED TO THE SECONDHANDS GITHUB.
```
Comment (michalkilawok) [inline SecondHandsY1PickAndPlace/README.txt]: please update this readme file

**Comment by felipebelo** [comment]
I would like if someone could have a look at this pull request in order to merge it? Thanks. @mirkow, @MuriloMartins, @michalkilawok

**Comment by mirkow** [comment]
@felipebelo Looking good so far.
What should work for now? The grasping of the first object, the wrench, fails on my pc because the grasp pose is unreachable and in the table.

Also the flashlight is wiggling sometimes due to bullet "features".
I replaced the table collision model with a primitive and lowered the CoM of the flashlight. This makes it better on my PC. I'll create a pull request.

**Comment by mirkow** [comment]
Ah, I see now that there is also a pull request in the Simulation Scenarios package, which might address that.

**Comment by felipebelo** [comment]
@mirkow @michalkilawok @vahrenkamp Could you please have a look that this is ok and accept the pull request if so?

**Comment by michalkilawok** [comment]
Looks like there's no .travis.yml file. Do we care about adding it?

**Comment by felixduvallet** [comment]
I suggest we don't block on adding a Travis file before closing this PR.

**Comment by felipebelo** [comment]
@felixduvallet or @michalkilawok: if one of you tested this and believe this is ok, could you please accept and merge my pull request?

**Pull request merged**